velox/dwio/parquet/RegisterParquetReader.h

```diff
@@ -18,10 +18,10 @@
 
 namespace facebook::velox::parquet {
 
-enum class ParquetReaderType { DUCKDB, NATIVE };
```

Comment:
We don't need this enum.

Reply:
I removed the enum altogether and updated the methods.